bundles/org.openhab.binding.lgwebos/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -15,6 +15,10 @@
 			<label>Access Key</label>
 			<description>Key exchanged with TV after pairing.</description>
 		</parameter>
+		<parameter name="macAddress" type="text" required="false">
+			<label>MAC Address</label>
+			<description>If MAC Address of TV is entered here, the binding will attempt to power on the device via Wake On Lan (WOL), when it receives command ON on channel power.</description>
```

Comment:
Please include mac address format in the description.

Reply:
done